resources/Materials/Examples/Syntax/PreFlattenedGraph.mtlx

```diff
@@ -0,0 +1,393 @@
+<?xml version="1.0"?>
+<materialx version="1.38" colorspace="lin_rec709" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
I copied over the original nodedefs + graphs and renamed them. Otherwise the same as the original test.
1. surfacematerial -> node instance of nodedef wrapping a standard surface
2. node insttance of nodedef referencing upstream nodegraph
3. nodegraph containing multiple nodes which are flattened. The outputs of each node are interdepedent.